src/gsheets/gsheets_utils.js

```diff
@@ -47,9 +47,24 @@ const addSheet = async (oAuth, sheetName, indexOffset=0) => {
     return processRequests(oAuth, requests)
 }
 
+// Clear values from Sheet
+const emptySheet = async (oAuth, sheetName, range) => {
+    const sheets = google.sheets({version: 'v4', auth: oAuth});
+    try {
+        return await sheets.spreadsheets.values.clear({
+            spreadsheetId: spreadsheet,
+            range: sheetName + '!' + range
+        })
+    } catch(err) {
+        console.log('[getValues] The API returned an error: ' + err);
+        return undefined
+    }
+}
+
 // Dump values to Sheet
 const updateValues = async (oAuth, sheetId, range, values) => {
     const sheets = google.sheets({version: 'v4', auth: oAuth});
+    await emptySheet(oAuth, sheetId, range) 
```

Comment:
I believe this is what I was looking for.
Clear(range) then DumpTo(range)

Reply:
I had noticed that previous commits were calling:
1. emptySheet()
2. updateValues()

Thanks for implementing this at the main entry point. 
Seems like a great place for it! Awesome

Reply:
Yep, was taking the issue step by step testing in different places and only in the end when I saw the bigger picture I realized that we could simplify it